bridge/bridgeapp/templates/registration/register.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'home.html' %} {% block main %}
+<section>
+    <form method="post" action="{% url 'register' %}">
+        {% csrf_token %} 
+
+        {% if form.errors %} 
+        <p>There are errors in the form!</p>
+        {% endif %} 
+
+        {{ form }}
+
+        <input type="submit" value="Register">
+    </form>
+</section>
+{% endblock %}
```

Comment:
I would think this page should go together with the `login.html` page, for the same reason stated in the comment concerning your code about the `login.html` page.

Also please consider how to code the front-end for the content that come with this built in form--there are numerous messages that if not handled stylistically would make this part of the app primitive.